adminSiteClient/CreateDataInsightModal.tsx

```diff
@@ -697,7 +783,31 @@ function NarrativeChartSelect({
                     label: name,
                 }))}
                 onSelect={handleSelect}
-                allowClear={true}
+                allowClear
+            />
+        </FormField>
+    )
+}
+
+function TopicTagsSelect({
+    allTopicTags,
+    show,
+}: {
+    allTopicTags: MinimalTag[]
+    show?: boolean
+}) {
+    if (!show) return null
+    return (
+        <FormField label="Topic tags" name="topicTagIds">
+            <Select
+                placeholder="Select topic tags..."
+                mode="multiple"
+                showSearch
```

Comment:
Oops, looks like this isn't working


https://github.com/user-attachments/assets/a2939c7a-8be3-4853-bf91-45e35aa96b70

Reply:
Good catch 👍🏻